Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -198,8 +198,11 @@ inline void
 SeedFinder<external_spacepoint_t, platform_t>::getCompatibleDoublets(
     Acts::SpacePointData& spacePointData,
     const Acts::SeedFinderOptions& options,
-    const Acts::SpacePointGrid<external_spacepoint_t>& grid,
-    boost::container::small_vector<Neighbour<external_spacepoint_t>, 9>&
+    const Acts::CylindricalSpacePointGrid<external_spacepoint_t>& grid,
+    boost::container::small_vector<
+        Acts::Neighbour<Acts::CylindricalSpacePointGrid<external_spacepoint_t>>,
```

Comment:
Another possibility here would be 

```suggestion
        Acts::Neighbour<decltype(grid)>,
```

Reply:
good idea ... this makes me thing we may template the class on the grid type in the short future